logstash-core/lib/logstash/pipeline.rb

```diff
@@ -327,6 +337,8 @@ def filter_batch(batch)
     # plugin.
     @logger.error("Exception in pipelineworker, the pipeline stopped processing new events, please check your filter configuration and restart Logstash.",
                   "exception" => e, "backtrace" => e.backtrace)
+
+    # TODO: shouldn't that be: "raise e" to re-raise previous exception??
```

Comment:
`raise` with no arguments will re-raise the previous exception:

```ruby
def oops
  raise "Oh no!"
end

begin
  oops
rescue => e
  raise
end
```

```
% ruby test.rb
test.rb:2:in `oops': Oh no! (RuntimeError)
        from test.rb:6:in `<main>'
```


From the ruby docs on `raise`:
> With no arguments, raises the exception in $! or raises a RuntimeError if $! is nil